plugin

```diff
@@ -250,6 +250,7 @@
   "Sese-Schneider/ha-energy-overview-card",
   "Shreyas-R/lovelace-wallpanel-screensaver",
   "Sian-Lee-SA/honeycomb-menu",
+  "silentbil/silent-bus",
```

Comment:
This looks like its created for a specific country, but the country is missing from the hacs.json file https://github.com/silentbil/silent-bus/blob/main/hacs.json

Reply:
Thanks, fixed